apps/meteor/app/action-links/server/lib/actionLinks.ts

```diff
@@ -14,11 +15,11 @@ function getMessageById(messageId) {
 
 // Action Links namespace creation.
 export const actionLinks = {
-	actions: {},
-	register(name, funct) {
+	actions: {} as { [key: string]: Function },
```

Comment:
should we type this `Function` as well? not sure how much work that would add 😂 

I saw at least the first param is `IMessage`, but other idk

Reply:
typed the func 👀 used the typings from the client file, but not really sure why `instance` is typed as undefined (and not sure what instance should be anyways)